atmos_model.F90

```diff
@@ -1747,8 +1765,9 @@ subroutine assign_importdata(rc)
                 do i=isc,iec
                   nb = Atm_block%blkno(i,j)
                   ix = Atm_block%ixp(i,j)
-                  if (IPD_Data(nb)%Sfcprop%oceanfrac(ix) > zero) then
-                    IPD_Data(nb)%Coupling%tisfcin_cpl(ix) = datar8(i,j)
+                  if (IPD_Data(nb)%Sfcprop%oceanfrac(ix) > zero .and.  datar8(i,j) > 150.0) then
```

Comment:
@SMoorthi-emc: For which case (eg., cmeps + frac_grid=T) did you find this and the change LN 1788 to be required?